src/main/kotlin/uk/gov/dluhc/notificationsapi/mapper/SourceTypeMapper.kt

```diff
@@ -1,24 +1,66 @@
 package uk.gov.dluhc.notificationsapi.mapper
 
-import org.mapstruct.InheritInverseConfiguration
-import org.mapstruct.Mapper
-import org.mapstruct.ValueMapping
+import org.springframework.context.MessageSource
+import org.springframework.stereotype.Component
+import uk.gov.dluhc.notificationsapi.dto.LanguageDto
+import uk.gov.dluhc.notificationsapi.messaging.models.SourceType
 import uk.gov.dluhc.notificationsapi.database.entity.SourceType as SourceTypeEntityEnum
 import uk.gov.dluhc.notificationsapi.dto.SourceType as SourceTypeDtoEnum
 import uk.gov.dluhc.notificationsapi.messaging.models.SourceType as SourceTypeMessageEnum
 import uk.gov.dluhc.notificationsapi.models.SourceType as SourceTypeApiEnum
 
-@Mapper
```

Comment:
I couldn't find a way to pass the messageSource in mapper/SourceTypeMapperTest.kt while using Mapstruct, so I've just gotten rid of Mapstruct. This is how all mappers are done in the search API so don't think it's an issue.